components/partials/header.js

```diff
@@ -25,11 +25,11 @@ export default function Header() {
               <li className="my-2 ml-0 sm:my-0 p-0">
                 <Link
                   href="/"
-                  className={
+                  className={`relative mr-[20px] sm:mr-[40px] text-[1rem] font-normal leading-[1.125rem] tracking-[1px] md:text-[1.0625rem] md:leading-[1.5rem] lg:text-[1.125rem] lg:leading-[1.875rem] after:absolute after:top-[29px] after:md:top-[24px] after:bottom-0 after:left-0 after:w-2/4 after:h-[2px] after:bg-black-900 ${
```

Comment:
I think there should be text size of all together like text-[1rem] md:text-[1.0625rem] and like that for more readability